### ISSUE TITLE
Please add the title length feature to the posts widget.Very important

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#15290]
It would be great if you could add a feature so that we can choose the length value for the post titles. This feature is available for posts, but not for titles.
Please add this feature. Thanks

**Comment by d19dotca** [comment]
I suspect restricting the title of a post in the Posts widget may negatively impact SEO. I'd think titles should never be shortened artificially and should just be changed in the actual post name if they're too long or too wordy.

**Comment by 79ho3ein** [comment]
@d19dotca 
I doubt it. Because this feature is currently available in the jet woobuilder plugin.

**Comment by d19dotca** [comment]
Just because it exists elsewhere though doesn't mean it's good for SEO, it just means there's a use-case for it. I'd be very cautious of using that feature myself. If a title is too long to be presented in a space constrained area, I'd think it'd be better to just adjust the title properly or to change the design of the listing of posts to accomodate long titles. Having an incomplete title presented in a list of posts doesn't seem too helpful to users and I suspect has a chance of negatively impacting SEO since Google won't know what the full title is when it's not in the text/HTML of the page listing all the posts.

**Comment by 79ho3ein** [comment]
Yes, thank you

**Comment by Aviad-Herman** [comment]
Hey,
I'm marking this as a feature suggestion, meantime you can use custom css.

I'm closing this issue and I'll open a ticket in our internal system (ED-3751). Note that I cannot commit 100% when / if we will do this.

```
selector h1, h2, h3, h4, h5, h6 {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}
```
![image](https://user-images.githubusercontent.com/73067562/122117704-d6f38380-ce2f-11eb-9b74-33c99089a23b.png)
